modules/simple_away.cpp

```diff
@@ -141,6 +150,15 @@ class CSimpleAway : public CModule {
 		PutModule("Timer disabled");
 	}
 
+	void OnMinClientsCommand(const CString& sLine) {
+		if (sLine.Token(1).empty()) {
+			PutModule(CString(m_iMinClients));
+		} else {
+			SetMinClients(sLine.Token(1).ToUInt());
+			PutModule("MinClients set");
```

Comment:
The existing module returns feedback with the number of seconds it would set to, for consistency, I think it would be nice to have this too return the number of clients it was set to. Like my previous comment, it would just add to the polish, but overall not change a whole lot.